src/main/java/com/airhacks/afterburner/views/QabelFXMLView.java

```diff
@@ -2,15 +2,30 @@
 
 import javafx.scene.Parent;
 
+import java.io.IOException;
+import java.util.Locale;
+import java.util.ResourceBundle;
 import java.util.function.Function;
 
-public abstract class QabelFXMLView extends FXMLView {
+public class QabelFXMLView extends FXMLView {
+
+
 	public QabelFXMLView() {
 		super();
+		this.bundle = ResourceBundle.getBundle("ui", calculateLocale(), new UTF8Converter());
 	}
 
 	public QabelFXMLView(Function<String, Object> injectionContext) {
 		super(injectionContext);
+		this.bundle = ResourceBundle.getBundle("ui", calculateLocale(), new UTF8Converter());
+	}
+
+	private Locale calculateLocale() {
```

Comment:
according to the [documentation](https://docs.oracle.com/javase/tutorial/i18n/resbundle/concept.html) this is not neccessary.

Name the property file after the language and not the locale, and it will work by magic.
Additionally, that would make the intent clearer because we don't really care about the locale but about the language.